core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -94,6 +96,10 @@ protected boolean isValidIdentifier(TableIdentifier tableIdentifier) {
     return true;
   }
 
+  protected Map<String, String> properties() {
+    return Collections.emptyMap();
```

Comment:
Nit: If it's empty, usually we use `ImmutableMap.of()`. I see you updated from `ImmutableMap.builder` so I assume you need null values, but I think even still we'd normally use `ImmutableMap.of()` in the empty case.